util/README.md

```diff
@@ -1,38 +1,35 @@
-# Convert Excel sheet to JSON
+## Convert Excel Sheet to JSON
 
-## Usage
+### Overview
 
-`convert_xlsx_to_json.js` is used to convert excel to JSON.
-`convert_json_to_data.js` has the transformation code to adhere to the required JSON format.
+The `convert_xlsx_to_json.js` script converts an Excel file to JSON format. The `convert_json_to_data.js` script transforms the JSON data to adhere to the required format.
 
-_`data/all_data.json` and `data/all_data.json` will be modified as per the new excel_
+### Usage
 
-### Note
-
--   Make sure excel file name is `report_<campus>.xlsx`
-
--   The sheet name in the file MUST be `Sheet 1`
+-   The scripts modify `data/all_data.json` as per the new Excel data.
+-   Ensure the Excel file is named in the format `report_<campus>.xlsx`.
+-   The sheet name must be `Sheet 1`.
```

Comment:
**Usage** doesn't make much sense here, remove the first point and move the remaining two into **Requirements**.

```md
### Requirements

-   Ensure the Excel file is named in the format `report_<campus>.xlsx`.
-   The sheet name must be `Sheet 1`.
-   The Excel sheet should contain the following column headers (case sensitive):

    | CODE | TITLE | TYPE | CREDITS | SLOT | FACULTY | VENUE |
    | ---- | ----- | ---- | ------- | ---- | ------- | ----- |
```